src/modules/registrypreview/RegistryPreviewUI/Strings/en-US/Resources.resw

```diff
@@ -118,7 +118,7 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="EditButton.Label" xml:space="preserve">
-    <value>Edit file...</value>
+    <value>Edit</value>
```

Comment:
Might be ancient guidelines, but if the current window opened a new window - modal or not - it was supposed to have the ... on the text command.  If no longer best practice, plz ignore

Reply:
The rule for ellipses is "Indicate a command that needs additional information (including confirmation) by adding an ellipsis at the end of the button label."
Opening a window/app is just a command 🙂